dandi/dandiapi.py

```diff
@@ -1,3 +1,27 @@
+"""
+Example code for downloading all assets from all published Dandisets that have
+"two-photon" in their ``metadata.measurementTechnique[].name``:
+
+.. code:: python
+
+    from pathlib import Path
+    from dandi.dandiapi import DandiAPIClient
+
+    with DandiAPIClient.for_dandi_instance("dandi") as client:
+        for dandiset in client.get_dandisets():
+            for version in dandiset.get_versions():
+                ds = dandiset.for_version(version)
```

Comment:
let's not go through all versions, but rather consider only released dandisets and take the latest released version and not include it in the resultant download path -- user would not need all versions (possibly identical) of the same file

Reply:
Done.